### DISCUSSION TITLE
How to auto log in log into my runescape account without having to open the jagex launcher everytime?

### DISCUSSION THREAD

**Discussion by skawngur-clash** [runelite/runelite#18366] [Q&A] [answered]
I've got everything working, only that everytime I try starting up RuneLite first, it makes me go back to jagex, and then I'm able to log in. I was just wondering if there is a way to bypass this so I can automatically log in just by starting up RuneLite

**Comment by cdfisher** [comment, marked as answer]
If you have a Jagex account (which you do if you've been using the Jagex launcher to sign in), you have to use the Jagex launcher to launch RuneLite.

**Reply by Koozwad** [reply]
I don't believe that. I mean, RuneLite through Jagex launcher is doing something different than regular RuneLite. We just need to find out what different commands it's using compared to usual and then add the commands to regular RuneLite batch script/command line.
